src/views/Dashboard.vue

```diff
@@ -203,6 +203,8 @@ export default {
             .then((resp) => {
               this.list = this.autoHideDetails(bars, true)
               this.members = resp.data.members
+              // ensure all members have a bar_id associated, else set default
+              this.members.map(m => { return m.bar_id ? m : Object.assign(m, { bar_id: this.community.default_bar_id })});
```

Comment:
I see you're trying to do such thing (ensuring the members including the CM have a bar assigned), but I guess it's not working properly ? Or am I getting a different but related error?

Reply:
The error was unrealted to this (see above).

Things should be working properly now